test/suites-ostree/isolated-network.robot

```diff
@@ -0,0 +1,79 @@
+*** Settings ***
+Documentation       Tests related to MicroShift running in an isolated network
+
+Resource            ../resources/ostree-health.resource
+Resource            ../resources/microshift-process.resource
+Resource            ../resources/microshift-host.resource
+Library             Collections
+
+Suite Setup         Setup
+Suite Teardown      Teardown
+
+Test Tags           network
+
+
+*** Variables ***
+${USHIFT_HOST}      ${EMPTY}
+${LB_CONFIG}        assets/isolated-lb-service.yaml
+${LB_NSPACE}        openshift-ingress
+${LB_SRVNAME}       isolated-lb-service
+${LB_PORTNUM}       31111
+
+
+*** Test Cases ***
+Verify Healthy Then Clean And Reboot MicroShift
+    [Documentation]    Verify no Internet access, check MicroShift health, clean
+    ...    MicroShift data and verify health again after rebooting the host
+
+    Verify No Internet Access
+    Wait For Healthy System
+
+    Cleanup MicroShift
+    Enable MicroShift
+    Reboot MicroShift Host
```

Comment:
Should we verify something after the reboot? Or is it a cleanup procedure before next test case?

Reply:
Good point, we should make sure that MicroShift comes back up and is healthy.

Reply:
It is a cleanup procedure + making sure that embedded images "remain" after clean. 
The next test starts with Wait for Healthy System, which is the verification we need.

Reply:
Test order is randomized, so we can't ensure that the tests run in the order written in the file. If the tests depend on specific pre-conditions, those need to be handled in the setup (either for the test or the suite).